specifyweb/attachment_gw/views.py

```diff
@@ -288,5 +297,62 @@ def proxy(request):
         (chunk for chunk in response.iter_content(512 * 1024)),
         content_type=response.headers['Content-Type'])
 
+@login_maybe_required
+@require_http_methods(['GET', 'POST'])
+def datasets(request, base_id: int):
+    with transaction.atomic():
+        base_resource = get_object_or_404(Spappresource, pk=base_id)
+        data = base_resource.spappresourcedatas.get()
+        datasets = list(json.loads(data.data))
+
+        if request.method == 'GET':
+            attrs = ('id', 'name', 'status', 'timeStampCreated', 'timeStampModified')
```

Comment:
Oh I thought that it would be better for `timeStamp` because camel case is what we use in the frontend.

Reply:
you are right.
I used to do that too - whenever there is a compound noun, I would capitalize each part
the benefit of doing that is than there is no question of whether a given compound noun part should be capitalized or not - you simply always capitalize it.
but the drawback is that the word is not semantically correct.
I don't know which way is better, so do whatever you feel is better/easier (or don't change it)

Reply:
a related part of this is that when I was less proficient with English than I am today, I didn't know whether a given word is a compound noun or two separate words, so tried to err on the side of camelCase over lowercase - this might be the case for others learning the language too

Reply:
fixed. Use timestampModified/Created now